include/ghex/bulk_communication_object.hpp

```diff
@@ -21,13 +21,47 @@
 #include "./common/utils.hpp"
 #include "./communication_object_2.hpp"
 #include "./rma/locality.hpp"
-#include "./rma/range_traits.hpp"
 #include "./rma/range_factory.hpp"
 #include "./rma/handle.hpp"
 
 namespace gridtools {
 namespace ghex {
 
+// RMA overview
+// ============
+//
+// The GHEX direct memory access is implemented for threads/processes that share the same memory.
+// This means, that in-node communication can be accelerated by by-passing message passing which
+// is otherwise employed. In order for direct memory access to work, the memory must be prepared
+// and the addresses must be communicated to the remote counter parts. Note, that word remote 
+// signifies a process/thread on the same shared memory (in-node) which is writing into memory which
+// it does not own (put operation). Conversly, the word local is used for a process/thread that
+// owns a memory resource which it exposes for remote puts. Since GHEX does only use put operations
+// (and no get operations), the local and remote processes/threads are also denoted as source and
+// target: source is where the data comes from (remote) and target is where the data is put to
```

Comment:
Isn't the source "local" and the target "remote" in a `put` operation?

Reply:
You may be right there. My thinking was, local is where the data to be updated is. I could change all the classes names if you insist (I would rather not though :) )

Reply:
I thought this was just an error in the comment. I did not find confusing in other places. I would not insist...